app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] When a user signs up they should be required to confirm their email, but you did not add this feature. You are missing the `:confirmable` module here.

Reply:
Thanks for the thorough review @julie-ify 😅!